src/redux/images-constructor/constructor-basic/tests/constructor-basic.reducer.test.js

```diff
@@ -0,0 +1,14 @@
+import { setConstructorBasic } from '../constructor-basic.actions';
+import { mockConstructorModel } from '../../constructor.variables';
+import constructorBasic from '../constructor-basic.reducer';
+
+describe('reducer tests', () => {
+  it('should return default store', () => {
+    expect(constructorBasic({})).toEqual({});
```

Comment:
put test variable "state = {}" in variables, after that import here that variable and put in expect() and .toEqual()

Reply:
fixed